### PR TITLE
Build form controls with OOUI

### PR DESCRIPTION
**Pull request by edg2s**

### CODE REVIEW

File: includes.php [hunk 1]
```diff
@@ -14,12 +14,15 @@
...
 
+OOUI\Theme::setSingleton( new OOUI\WikimediaUITheme() );
+
 echo '<!DOCTYPE html>
 <html>
 	<head>
 		<meta charset="utf-8">
 		<title>Patch demo</title>
 		<link rel="stylesheet" href="index.css">
+		<link rel="stylesheet" href="node_modules/oojs-ui/dist/oojs-ui-wikimediaui.min.css">
```
Comment (edg2s) [inline includes.php:25]: we don't deploy with `npm i` at the moment, so perhaps this file should just be copied to a lib folder?

**Comment by MatmaRex** [comment]
This is another dependency that I don't really want to have and that someone will have to update regularly until the end of times. Eh…

File: includes.php (hunk 1 above)
Comment (edg2s) [reply includes.php:25]: Adding `npm install --production` to the deploy step is probably the easiest.

**Comment by MatmaRex** [comment]
Alright, fine, let's do this… Thank you for updating the patch.

I have some tweaks I want to make, I'll push them here for you to have a look.

**Comment by MatmaRex** [comment]
I think this should work, but apparently the `<summary>` tag stops being clickable when you put a `<label>` inside of it. That's a bit dumb.

I need to update the JS code for disabling checkboxes too.

I'll finish this tomorrow, bleh.

**Comment by MatmaRex** [comment]
Never mind, I finished it today. I think I like it, please have a look at my changes.

**Review by MatmaRex — APPROVED** [review]

**Comment by edg2s** [comment]
The span inside details is display:block:
image

**Comment by edg2s** [comment]
Fixed that, LGTM now.

**Comment by MatmaRex** [comment]
Thanks, I only tested on Firefox and I didn't see that.

Some screenshots for future reference:

Firefox:
| Before | After |
| -- | --|
|  | 
|  | 

Chromium:
| Before | After |
| -- | --|
|  | 
|  |

**Pull request merged**